data/homepage.yaml

```diff
@@ -131,10 +131,11 @@ contribute:
     - Code a new module or fix a bug.
     - Improve the documentation.
   main: >-
-    If you want to get involved, come talk to us.
+    Want to get involved?
+    Come talk to us.
     We're friendly.
     And we have contributor resources to help you.
-  link: https://forum.ansible.com/pub/how-to-contribute
+    Click <a href="https://forum.ansible.com/pub/how-to-contribute" target="_blank">here</a> to learn more!
```

Comment:
HTML in the YAMl feels weird....

Reply:
Yeah it was late at night. Way better to put this in a "learn more" section and keep section links in a consistent place.